### PR TITLE
Added EditorConfig and Travis CI test

### PR DESCRIPTION
**Pull request by kim-sondrup**

### CODE REVIEW

**Comment by seansan** [comment]
Does this need some testing? and then be applied?

**Comment by centerax** [comment]
which editor does support *EditorConfig*?

**Comment by kim-sondrup** [comment]
Some of the supported editors
GitHub, PhpStorm, Vim, Sublime Text, Notepad++, Emacs, Visual Studio

You can find an extended list on [editorconfig.org](http://editorconfig.org/#download)

**Comment by centerax** [comment]
great stuff @kim-sondrup Can you solve the conflicts so we can merge?

**Comment by kim-sondrup** [comment]
I have rebased it with the develop branch now

File: package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (centerax) [inline package.json]: do we need this file? This is a Magento extension not a node module.

Comment (kim-sondrup) [reply package.json]: Here `package.json` is a helper file for the developer to install packages needed to test their code. We could add a install command to `.travis.yml` like `npm install editorconfig-tools` but that kind of setup have been replaced once before when Composer dev requirements was handled that way e2484ce47b2fe1687b5ed6ab6fda3a8c13d804a4

**Pull request merged**